docs/application/get-started/iot.md

```diff
@@ -0,0 +1,18 @@
+# Get Started for IoT
+
+The Get Started Tutorial teaches you how to create basic Tizen applications for IoT devices.
+
+## .NET Application
+
+The following guide demonstrates how to create and run a basic Tizen .NET application, which displays some text on the screen with no user interaction:
+
+-   [IoT .NET application](../dotnet/get-started/iot/first-app.md)
+
+    An IoT application is targeted to run on an IoT device. The applications can take advantage of a wide variety of functionalities available on IoT devices.
+
+First, ensure that you have installed the latest version of Visual Studio Tools for Tizen. <a href="https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen" target="_blank"><img src="media/ic_docs_download.png"><Strong> Download Visual Studio Tools for Tizen</strong>.</a>
+
+
+
+## Hardware Setup for IoT Development
+To start developing Tizen IoT applications, follow the <a href="http://tizenschool.org/tutorial/137/" target="_blank"><strong> guide</strong></a> that demonstrates how to set up the Raspberry Pi board.
```

Comment:
> **bson1012** 
Regarding your second comment about the use of HTML tags, I just followed what other pages (mobile, tv and wearable) are doing. :)

Thanks for pointing out that html tags are used in other pages also.
We are guiding to use markdown format, so we should fix those pages also.
We will update the other pages.
Would you please apply Markdown tag at here and line 13?